tests/common/test_exception.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from aiohttp import web
+
+from ai.backend.common.exception import (
+    BackendAIError,
+    ErrorCode,
+    ErrorDetail,
+    ErrorDomain,
+    ErrorOperation,
+    InvalidAPIParameters,
+    UserNotFound,
+)
+
+
+class TestBackendAIErrorCode:
+    @pytest.mark.asyncio
+    async def test_error_code_in_body(self) -> None:
+        """Test that error_code() is correctly included in the error body."""
+        error = UserNotFound(extra_msg="User with ID 123 not found")
+
+        # Verify error_code is in the body_dict
+        assert "error_code" in error.body_dict
+        assert error.body_dict["error_code"] == str(error.error_code())
+
+        # Verify the error code format
+        error_code = error.error_code()
+        assert str(error_code) == str(error.error_code())
+
+    @pytest.mark.asyncio
+    async def test_multiple_instances_have_same_error_code(self) -> None:
+        """Test that different instances of the same error class return the same error code."""
+        error1 = InvalidAPIParameters(extra_msg="Missing parameter: foo")
+        error2 = InvalidAPIParameters(extra_msg="Missing parameter: bar")
+
+        # Both instances should return the same error code
+        assert error1.error_code() == error2.error_code()
+        assert str(error1.error_code()) == str(error2.error_code())
```

Comment:
This test doesn't seem necessary. It's ambiguous to call it an implementation requirement.